Controller/Component/WizardComponent.php

```diff
@@ -382,7 +382,7 @@ public function process($step) {
 					} else {
 						throw new NotImplementedException(sprintf(__('Process Callback not found. Please create Controller::%s', $processCallback)));
 					}
-					if ($proceed) {
+					if ($proceed === true) {
```

Comment:
Gotchas
When testing the return value of a function that can return either 0 or boolean false, like strpos(), always use === and !==, or you’ll run in to problems.